docs/docs/recipes.md

```diff
@@ -31,2103 +31,38 @@ See [docs templates](/docs/docs-templates/) in the contributing docs for more he
 
 Craving a happy medium between [full-length tutorials](/tutorial/) and crawling the [docs](/docs/)? Here's a cookbook of guiding recipes on how to build things, Gatsby style.
 
-## 1. Pages and Layouts
```

Comment:
Love this. Given this is now an overview page, we should hide the table of contents. I asked for clarification on that in #18822 (cc @gillkyle)!

Reply:
This is now documented here: https://www.gatsbyjs.org/contributing/docs-and-website-components/#table-of-contents